### PR TITLE
Header

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR contains 

-  Header component

### CODE REVIEW

**Review by chanchal16 — APPROVED** [review]
- your code looks clean and is readable.
- Good approach of using figure element for logo instead of div or span.
- liked that you made reusable component for button.
Good to go ahead!

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import { Router } from "./Router";
 
-import "./App.scss";
+import { Header } from "./component";
+
```
Comment (Ronak-Mutha) [inline src/App.js:4]: For consistency, you can remove the exta space here

**Review by Ronak-Mutha — APPROVED** [review]
Good use of semantic tags and responsive css units. Good to merge.

File: src/App.js (hunk 1 above)
Comment (junaidshaikh-js) [reply src/App.js:4]: Oh, I must have missed that. Thank you for catching it, Ronak. 

Changes are made and will be pushed in the next PR.

**Pull request merged**